### ISSUE TITLE
[gdb] DbgObject.locals does not work 

### ISSUE THREAD

**Issue by cbiesinger** [MicrosoftEdge/JsDbg#77]
Returns an empty array.

Looking at the verbose output, it looks like it works by calling GetCallStack followed by LookupSymbolName, and the latter does not currently work for stack variables

**Comment by cbiesinger** [comment]
OK, I used the function incorrectly but there are actual bugs here:

```js
> JsDbg.GetCallStack(20, (e) => { JsDbg.LookupLocalsInStackFrame(e[0].instructionAddress, e[0].stackAddress, e[0].frameAddress, (E) => console.log(E)) });
VM1227:1 {error: "Can't find stack frame"}

> JsDbg.GetCallStack(20, (e) => { JsDbg.LookupLocalsInStackFrame(e[1].instructionAddress, e[1].stackAddress, e[1].frameAddress, (E) => console.log(E)) });
VM1294:1 {error: "'gdb.Value' object has no attribute 'value'"}
```

**Comment by cbiesinger** [comment]
The problem with the first stack frame is fixed by https://github.com/MicrosoftEdge/JsDbg/pull/84

Still need to figure out the gdb.Value issue

**Comment by cbiesinger** [comment]
This was fixed by #84
